tools/debian/first_package_finder.py

```diff
@@ -0,0 +1,120 @@
+"""Functions to help find the first package version for a specific release"""
```

Comment:
As discussed let's add a main() to this and make this a standalone script that dumps the results in a JSON.